### PR TITLE
Refactor i18n Localization and Background Page Fetch

### PR DESCRIPTION
**Pull request by kabir-afk**
**Description:**
This Pull Request addresses the following changes:
1. **Fix:** Resolved an issue with i18n localization in SignItCoreContent.js to ensure compatibility with both Chrome and Firefox. The issue was referenced earlier at [#19](https://github.com/lingua-libre/SignIt/issues/19) where relevant challenges faced are mentioned. While I did try passing banana as a parameter , and its values are certainly passed , the i18n functionality isn't imported...in my commit earlier emphasized on the same but things only worked in Chrome...now they do in FF as well
2. **Update:** Updated the SVG icon URLs for previous, next, and search in popup.css for improved UI aesthetics.
3. **Refactor:** Streamlined the code to fetch the background page efficiently for better performance.

**Changes:**
- Refactored code in SignItCoreContent.js for i18n localization fix.
- Updated SVG icon URLs in popup.css for better icon display.
- Optimized code for background page fetching for enhanced efficiency.

**Commits:**
1. af51c9a - Fix issue with i18n localization in SignItCoreContent.js
2. 92a23e1 - Update SVG icon URLs in popup.css
3. 46339be - Refactor code to fetch the background page efficiently

### CODE REVIEW

**Review by hugolpz — APPROVED** [review]
Thank for this code improvement.

I assume `text` is temporarily broken and therefore commented. https://github.com/lingua-libre/SignIt/pull/65/commits/46339bee0609979aca8cd33f92eca5a84804f769#diff-2ce48ac6dbf8501b60a7bbeb4c05cbd8b5eead20f4566669b124ade52182387bR120-R126

**Review by hugolpz — APPROVED** [review]
Approving simple change on CSS SVG icone calls.

**Comment by kabir-afk** [comment]
> Thank for this code improvement.
> 
> I assume `text` is temporarily broken and therefore commented. [46339be#diff-2ce48ac6dbf8501b60a7bbeb4c05cbd8b5eead20f4566669b124ade52182387bR120-R126](https://github.com/lingua-libre/SignIt/commit/46339bee0609979aca8cd33f92eca5a84804f769#diff-2ce48ac6dbf8501b60a7bbeb4c05cbd8b5eead20f4566669b124ade52182387bR120-R126)

Yea, well actually `this.searchWidget` relied on records inside `_backgroundPage` which dont work apparently , so I commented them out temporarily

File: SignItCoreContent.js [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
+var banana,resArr;
 (async()=>{
-  const sourceMap = new Map(
-    await chrome.runtime.sendMessage({ command: "getBanana" })
-  );
-  banana = { i18n: (msg) => sourceMap.get("fr")[msg] };
+  resArr = await chrome.runtime.sendMessage({ command: "getBanana" });
+  console.log(resArr);
+  const sourceMap = new Map(resArr[0]);
+  banana = { i18n: (msg,locale) => sourceMap.get(locale)[msg] };
```
[the remark this answers is not in the document]
Comment (kabir-afk) [reply SignItCoreContent.js:8]: well that locale is different 😅 ....this is rather passed as argument , which is set as per the `sendResponse` inside `sw.js`. Although I forgot to introduce those changes of `sw.js` inside this PR..apologies

**Comment by hugolpz** [comment]
Understood. Thank for the explainer.

**Pull request merged**